content/api/commands/intercept.md

```diff
@@ -101,16 +106,36 @@ All properties are optional. All properties that are set must match for the rout
 | times      | Maximum number of times to match (`number`)                                                     |
 | url        | Full HTTP request URL                                                                           |
 
-#### <Icon name="angle-right"></Icon> routeHandler (<code>string | object | Function | [StaticResponse][staticresponse]</code>)
+See [examples](#With-RouteMatcher) below.
+
+#### <Icon name="angle-right"></Icon> staticResponse (<code>[StaticResponse][staticresponse]</code>)
+
+By passing in a `StaticResponse` as the last argument, you can [statically define (stub) a response](#Stubbing-a-response) for matching requests including the body of the response, as well as the headers and HTTP status code:
+
+| Option     | Description                                            |
+| ---------- | ------------------------------------------------------ |
+| statusCode | HTTP response status code                              |
+| headers    | HTTP response headers                                  |
+| body       | Serve a static string/JSON object as the response body |
+| fixture    | Serve a fixture as the HTTP response body              |
+
+`StaticResponse` also provides options for simulating a degraded or broken network connection:
```

Comment:
This is how it was presented initially and I'd like to keep it that way because one table defines the actual response content while the other defines how it will be delivered.